worker-message-prioritization-distribution/src/main/java/com/github/workerframework/workermessageprioritization/redistribution/shovel/ShovelDistributor.java

```diff
@@ -122,49 +129,45 @@ public ShovelDistributor(
-        try {
-            while(true) {
-                runOnce();
-
-                try {
-                    Thread.sleep(distributorRunIntervalMilliseconds);
-                } catch (final InterruptedException e) {
-                    LOGGER.warn("Interrupted {}", e.getMessage());
-                    throw e;
-                }
-            }
-        } finally {
-            try {
-                nonRunningShovelCheckerExecutorService.shutdownNow();
-            } catch (final Exception exception) {
-                LOGGER.warn("Failed to shutdown nonRunningShovelCheckerExecutorService", exception);
-            }
+        while(true) {
+            runOnce();
 
             try {
-                nonRunningShovelCheckerExecutorService.shutdownNow();
-            } catch (final Exception exception) {
-                LOGGER.warn("Failed to shutdown shovelRunningTooLongCheckerExecutorService", exception);
-            }
-
-            try {
-                corruptedShovelCheckerExecutorService.shutdownNow();
-            } catch (final Exception exception) {
-                LOGGER.warn("Failed to shutdown corruptedShovelCheckerExecutorService", exception);
+                Thread.sleep(distributorRunIntervalMilliseconds);
+            } catch (final InterruptedException e) {
+                LOGGER.warn("Interrupted {}", e.getMessage());
+                throw e;
             }
         }
     }
     
     public void runOnce() {
-        
-        final Set<DistributorWorkItem> distributorWorkItems = getDistributorWorkItems();
+
+        final Set<DistributorWorkItem> distributorWorkItems;
+        try {
+            distributorWorkItems = getDistributorWorkItems();
+        } catch (final Exception e) {
+            final String errorMessage = String.format(
+                    "Failed to get a list of distributor work items, so unable to check if any shovels need to be created. " +
+                            "Will try again during the next run in %d milliseconds",
+                    distributorRunIntervalMilliseconds);
+
+            LOGGER.error(errorMessage, e);
+
+            return;
+        }
+
+        if (shovelsApi.getApi().getShovels(rabbitMQVHost).stream().map(s -> s.getName()).anyMatch(s -> s.equals("rorytest"))) {
+            throw new RuntimeException("Rory temporarily testing JVM shutdown");
+        }
 
         final List<RetrievedShovel> retrievedShovels;
         try {
             retrievedShovels = shovelsApi.getApi().getShovels(rabbitMQVHost);
         } catch (final Exception e) {
             final String errorMessage = String.format(
-                    "Failed to get a list of existing shovels, so unable to check if additional shovels need to " +
-                            " be created or recreated. Will try again during the next run in %d milliseconds",
+                    "Failed to get a list of existing shovels, so unable to check if additional shovels need to be created. " +
```

Comment:
Updating this log message as it is out of date. This class used to recreate shovels as well as create them, but now it only creates them. Recreating shovels was moved to the *Checker classes that check for bad shovels.